include/drivers/pic.h

```diff
@@ -49,8 +49,8 @@
 #define PIC_CASCADE_PIC2_IRQ 0x02
 #define PIC_CASCADE_PIC1_IRQ 0x04
 
-#define PIC_IRQ0_OFFSET 0x20
-#define PIC_IRQ8_OFFSET 0x28
+#define PIC_IRQ0_OFFSET 0x28
```

Comment:
just a general comment: we really have to find a way to standardize irq vector numbers assignment... it will get messy pretty soon.